### ISSUE TITLE
Isn't a issue but a suggestion

### ISSUE THREAD

**Issue by vinilop1114** [daydreamer1222/WeCatch-Android#15]
It'll be nice if the app notify when a specific 100 IV pokemon appear.

**Comment by aidrouge** [comment]
Yes please dev 🙏 and add list of all coordinate for only filtered pokemon like with 100 90 80 0 Iv
